### ISSUE TITLE
About the pretrained weights result, can not performance well.

### ISSUE THREAD

**Issue by yicheng-2019** [SvipRepetitionCounting/TransRAC#39]
Thanks for your great job, TransRac!
Now I have some queastion about TransRac:
We runned the training of model on training set of Repcout-A dataset with default option. The result we get in test set MAE=0.59 OBO=0.29.
And we also downloaded the weight from the TranRac project Model zoo named "transrac_ckpt_171.pt", but get the result in test data is only MAE=0.73 OBO=0.17.
So the trained model from your Github record shows quite different result with your paper claim (MAE=0.44). 
Can you explain more detail about this problem? Or Is there any special operation we shall perform before run the test dataset?
Thanks for your time and reading. We are looking forward to your reply.

Our test environment：
OS：ubuntu20.04
GPU：Nvidia3090
Pytorch：1.10.1
cuda：11.3

**Test Code based on './test.py'**:
import os
from dataset.RepCountA_raw_Loader import MyData
from models.TransRAC import TransferModel
from testing.test_looping import test_loop

N_GPU = 1
device_ids = [i for i in range(N_GPU)]
os.environ["CUDA_VISIBLE_DEVICES"] = "0"

root_path = "/dataset/LLSP"

test_video_dir = 'test'
test_label_dir = 'test.csv'

config = './configs/recognition/swin/swin_tiny_patch244_window877_kinetics400_1k.py'
checkpoint = './pretrained/swin_tiny_patch244_window877_kinetics400_1k.pth'
lastckpt = "./transrac_ckpt_pytorch_171.pt"

NUM_FRAME = 64
SCALES = [1, 4, 8]
test_dataset = MyData(root_path, test_video_dir, test_label_dir, num_frame=NUM_FRAME)
my_model = TransferModel(config=config, checkpoint=checkpoint, num_frames=NUM_FRAME, scales=SCALES, OPEN=False)
NUM_EPOCHS = 1

test_loop(NUM_EPOCHS, my_model, test_dataset, lastckpt=lastckpt)

**Test Results:**
load checkpoint from local path: ./pretrained/swin_tiny_patch244_window877_kinetics400_1k.pth
--------- backbone loaded ------------
  0%|          | 0/1 [00:00<?, ?it/s]
  0%|          | 0/152 [00:00<?, ?it/s]
  1%|          | 1/152 [00:01<03:07,  1.24s/it]predict count :3.0, groundtruth :2.0

  1%|▏         | 2/152 [00:02<02:47,  1.12s/it]predict count :6.0, groundtruth :15.0
predict count :6.0, groundtruth :8.0

  2%|▏         | 3/152 [00:03<02:23,  1.03it/s]
  3%|▎         | 4/152 [00:04<02:29,  1.01s/it]predict count :8.0, groundtruth :12.0
 .....(omitted the middle processing)
100%|██████████| 152/152 [02:27<00:00,  1.03it/s]
100%|██████████| 1/1 [02:27<00:00, 147.74s/it]
predict count :1.0, groundtruth :13.0
MAE:0.7329175710482033,OBO:0.17763157894736842

**Comment by ZYQbill** [comment]
You can try to transfer the .mp4 to .npz for testing. This can reproduce the similar results in the paper, you can refer to other similar issues.
